evalai/submissions.py

```diff
@@ -114,32 +127,44 @@ def push(image, phase):
     federated_user = response["success"]["federated_user"]
     repository_uri = response["success"]["docker_repository_uri"]
 
-    AWS_ACCOUNT_ID = federated_user["FederatedUser"]["FederatedUserId"].split(
-        ":"
-    )[0]
-    AWS_SERVER_PUBLIC_KEY = federated_user["Credentials"]["AccessKeyId"]
-    AWS_SERVER_SECRET_KEY = federated_user["Credentials"]["SecretAccessKey"]
-    SESSION_TOKEN = federated_user["Credentials"]["SessionToken"]
+    # Default Environment
```

Comment:
Please change this to  `Production Environment`

Reply:
Done